src/nomad_material_processing/__init__.py

```diff
@@ -94,7 +96,7 @@ class Parallelepiped(Geometry):
         a_eln=ELNAnnotation(
             component=ELNComponentEnum.NumberEditQuantity,
             defaultDisplayUnit='millimeter',
-            label='Width (x)',
+            label='Width (x or a)',
```

Comment:
What is the idea of adding a, b, and c? I would stick with one of the notations, this becomes quite messy.

Reply:
switched back to x-y-z